_posts/2017-01-17-announcing-afip-bill.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "Announcing AfipBill"
+date: 2016-12-14 18:06:00
+categories: ["open-source", "rubygems"]
+author: "luciano"
+---
+
+If you live in Argentina and you ever use AFIP, you should already know that their platform is not the best in terms of user friendly.
+One of the things that we wanted to do for [OmbuShop](https://www.ombushop.com) was to access AFIP via [API](http://www.afip.gob.ar/ws) in order to print the bills of each seller. Unfortunately, there is no way to do this because the API doesn't have that kind of scope.
```

Comment:
@lubc it sounds better if you say "we wanted to integrate OmbuShop with AFIP (using their API) in order to print ... " and "... in order to generate and print the bills for each seller"

And then "... because the API doesn't generate a printable version (PDF) of the bill"